src/services/blocks/BlocksService.ts

```diff
@@ -243,13 +236,15 @@ export class BlocksService extends AbstractService {
 				 * https://github.com/polkadot-js/api/issues/2365
 				 */
 				let extrinsicBaseWeight;
-				if (
-					specName !== api.runtimeVersion.specName.toString() ||
-					specVersion !== api.runtimeVersion.specVersion.toNumber()
-				) {
-					// We are in a runtime that does **not** match the decorated metadata in the api,
-					// so we must fetch the correct metadata, decorate it and pull out the constant
-					const decorated = expandMetadata(api.registry, metadata);
+				if (runtimeDoesNotMatch) {
+					if (!decorated) {
+						extrinsics[idx].info = {
```

Comment:
I am realizing I think we will still want the `RuntimeDispatchInfo` even if we can't calculate the partial fee. We are not planning on supporting fee calcs generically, but we should still support including the dispatch class and weight if possible. Originally, `calc_fee` just returned `null` for other chains, so the dispatch info would be 
```
{
    weight: 100,
    class: Normal,
    partialFee: null
}
```
I honestly forgot about this and I probably introduced a regression when I added in 
```
				if (!extrinsicBaseWeight) {
					throw new InternalServerError(
						'`extrinsicBaseWeight` is not defined when it was expected to be defined. File an issue at https://github.com/paritytech/substrate-api-sidecar/issues'
					);
				}
```
in the last PR. But I think going forward we should try to ensure we can deliver the runtime dispatch info as above and have a test to make sure a regression does not happen